glslc/src/main.cc

```diff
@@ -200,6 +310,44 @@ int main(int argc, char** argv) {
       }
     } else if (arg == "-fauto-bind-uniforms") {
       compiler.options().SetAutoBindUniforms(true);
+    } else if (arg == "-fhlsl-iomap") {
+      compiler.options().SetHlslIoMapping(true);
+    } else if (arg == "-fimage-binding-base") {
```

Comment:
That's a pretty cool trick.   I thought about making a lookup table, but your technique is better.  I'll try it out and see if I like it. For another CL though.